mirrorlist/mirrorlist_client.wsgi

```diff
@@ -179,8 +179,70 @@ def application(environ, start_response):
     return response(environ, start_response)
 
 
-if __name__ == '__main__':
-    from wsgiref import simple_server
-    httpd = simple_server.make_server('127.0.0.1', 8090, application)
-    print('Serving on http://127.0.0.1:8090')
-    httpd.serve_forever()
+if __name__ == '__main__':  # pragma: no cover
+    import sys
+    import os
+
+    # Run the unit tests. You can also perform coverage analysis by running:
+    # coverage run mirrorlist_client.wsgi --test && coverage html
+    if '--test' in sys.argv:
```

Comment:
For the ``.wsgi`` script?  Nah.  It is almost never directly distributed to users.  Usually either run by apache or by a developer automatically in the tests.